Good-Bids/goodbids
# [#] Prettier Config

# Summary

- Fixes format and format:fix not finding format mismatches
- Hooks prettier-plugin-tailwindcss in to config
- Fixes a whole bunch of tailwind mismatches

## Testing Instructions

1. Make some invalid format changes
2. Run `npm run format` to see 'em
3. Run `npm run format:fix` to fix 'em
